### DISCUSSION TITLE
PostgreSQL version?

### DISCUSSION THREAD

**Discussion by yoyoungeryo** [akiver/cs-demo-manager#769] [Q&A] [answered]
![image-20240222130946985](https://github.com/akiver/cs-demo-manager/assets/96250166/3e4c18d0-74f3-4bde-89b8-3c147aed3cb9)

+ it just say there is a Grammatical errors around      "("
please if anybody could help me.

**Comment by yoyoungeryo** [comment]
and i use postgresql 11

**Comment by akiver** [comment, marked as answer]
PostgreSQL 11 is not supported, you have to install the last version.

**Reply by NoPlagiarism** [reply]
Then, please, mention minimal or at least recommended PostgreSQL version in User Manual

**Reply by yoyoungeryo** [reply]
thanks i have saved it

**Comment by yoyoungeryo** [comment]
> PostgreSQL 11 is not supported, you have to install the last version.

i have tried to install postgresql 16 the last version ,but it can not install on my computer
![zhege](https://github.com/akiver/cs-demo-manager/assets/96250166/08735d9e-d07e-4503-9ace-5217e5f8f8e3)

**Reply by yoyoungeryo** [reply]
:(,,,,
what is the problem?
